R/new_functions.R

```diff
@@ -433,29 +433,55 @@ observation.experiment <- function(x) {
 #' Produces an \code{experiment} object from the replication of an
 #' \code{experiment} object.
 #'
-#' @param x An object of class \code{experiment}.
-#' @param n The number of times to repeat the experiment \code{x}.
+#' Parameters \code{times} and \code{exp} are exclusive.
+#' If both specified, \code{times} will be taken.
+#'
+#' @param exp An object of class \code{experiment}.
+#' @param times Number of replicates
+#' @param by.rows A vector of rows to propagate in \code{exp}.
 #'
 #' @return An object of class \code{experiment} that is the replication of the
 #' inputed \code{x} argument.
 #'
 #' @export
-repl <- function(x, n) UseMethod("repl")
+repl <- function(exp, times = NULL, by.rows = NULL) UseMethod("repl")
```

Comment:
You don't need 2 arguments `times` and `by.rows` here. Please combine them into one single argument that can either be

- a named vector or
- a single unnamed value

Reply:
Done